test/performance/logging.cc

```diff
@@ -196,7 +196,7 @@ TEST_P(LoggingTest, RunThreads)
 }
 
 INSTANTIATE_TEST_SUITE_P(LoggingTest, LoggingTest,
-                         ::testing::Values(1, 2, 4, 8, 16, 32));
+                         ::testing::Values(1, 2, 4, 8, 16));
```

Comment:
these changes will conflict with #302

Reply:
Let's get the forward port done first, and then I will sort it out.

Reply:
this is close; I'll just resolve the conflicts when merging forward

Reply:
forward port is done, feel free to resolve conflicts